build.gradle

```diff
@@ -216,6 +214,10 @@ android {
     lint {
         abortOnError false
     }
+    buildFeatures {
+        renderScript true
+        aidl true
+    }
```

Comment:
this addition added by some automation? otherwise (or in general :) - what does it mean?

Reply:
yes, this was done automatically by AndroidStudio, it shows a dialog saying that your gradle plugin version is behind and offers to upgrade it, then it does that changes automatically, porting to the new config idioms of the new version it seems and some settings get deprecated etc